docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Docker Compose file for FreeCodeCamp
+#
+# Bootstrap with:
+#   docker-compose run --rm server npm install
+#   docker-compose run --rm server npm run only-once
+#
+# Run with:
+#   docker-compose up
+#
+version: "3"
+services:
+    db:
+        image: mongo:3.2.6
+        ports:
+            - "27017:27017" # Change port if needed
+    freecodecamp:
+        image: node:8.9.4
+        depends_on:
+            - db
+            - mailhog
```

Comment:
Added mailhog as a dependency.

Reply:
Can you expose the mailhog web port? `8025` off the top of my head, might be wrong.

Reply:
Yup, its `8025`

Reply:
Sure!